CWE/decisions/z MedPac.txt

```diff
@@ -0,0 +1,110 @@
+			POR = { exists = yes OR = { government = populist_dictatorship government = nationalist_dictatorship } is_vassal = no } 
+			SPA = { exists = yes OR = { government = populist_dictatorship government = nationalist_dictatorship government = absolute_monarchy } is_vassal = no }
+			FAG = { exists = yes OR = { government = populist_dictatorship government = nationalist_dictatorship } is_vassal = no } 
+			GRE = { exists = yes OR = { government = populist_dictatorship government = nationalist_dictatorship government = absolute_monarchy } is_vassal = no } 
+			ALB = { exists = yes OR = { government = hms_government government = hms_government1 government = absolute_monarchy } is_vassal = no } 
+			CYP = { exists = yes government = populist_dictatorship is_vassal = no } 
+			}
+		}
+		effect = {
+		prestige = 100
+		money = -100000
+        set_global_flag = form_MedPac
+        add_country_modifier = { name = alliance_MedPac duration = 1825 }
+		}
+		ai_will_do = { 
+			factor = 1 
+		}
+	}
+
+	join_MedPac = {
+		picture = "MedPac"
+		potential = {
+
+            OR = {
+			    ITA = { exists = yes OR = { government = populist_dictatorship government = nationalist_dictatorship government = presidential_dictatorship } is_vassal = no } 
+			    POR = { exists = yes OR = { government = populist_dictatorship government = nationalist_dictatorship } is_vassal = no } 
+			    SPA = { exists = yes OR = { government = populist_dictatorship government = nationalist_dictatorship government = absolute_monarchy } is_vassal = no }
+			    FAG = { exists = yes OR = { government = populist_dictatorship government = nationalist_dictatorship } is_vassal = no } 
+			    GRE = { exists = yes OR = { government = populist_dictatorship government = nationalist_dictatorship government = absolute_monarchy } is_vassal = no } 
+			    ALB = { exists = yes OR = { government = hms_government government = hms_government1 government = absolute_monarchy } is_vassal = no } 
+				CYP = { exists = yes government = populist_dictatorship is_vassal = no } 
+		    }
+
+			NOT = { has_country_modifier = alliance_MedPac }
+			NOT = { has_country_modifier = warsaw_pact }
+			NOT = { has_country_modifier = csto_alliance }
+			NOT = { has_country_modifier = alliance_sato }
+			NOT = { has_country_modifier = rio_pact }
+			
+		    NOT = { has_country_modifier = leave_alliance_MedPac }
+			has_global_flag = form_MedPac
+		}
+		allow = {
+			war = no
+			is_vassal = no
+		}
+		effect = {
+            any_country = {
+                limit = { has_country_modifier = alliance_MedPac }
+                relation = { who = THIS value = 100 }
+                create_alliance = THIS
+            }
+
+            add_country_modifier = { name = alliance_MedPac duration = 1825 }
+		}
+		ai_will_do = { 
+			factor = 1 
+		}
+	}
+
+	leave_MedPac = {
+		picture = "MedPac"
+		potential = {
+			has_country_modifier = alliance_MedPac
+		}
+		allow = {
+			war = no
```

Comment:
The allow conditions for the leave decision are exactly the same as those for the join decision. This will cause an infinite loop between the decisions.